docs/modules/operation/pages/topology/asset-topology-provider/view-additional-notes.adoc

```diff
@@ -0,0 +1,16 @@
+== View the topology
+
+After installing the topology and refreshing your screen, you should see a new topology display option in the {page-component-title} topology page.
+The label field provides the displayed name of this topology.
+
+The label field does not need to be the same as the providerId that the REST API uses for the installation or removal of a topology.
+However, the label field must be unique across all installed topologies.
+
+It is possible to have several topologies installed that have been generated using different configurations.
+You must ensure that the providerId and label field used for each installation command are different.
+
+== Additional notes
+
+You must first uninstall a {page-component-title} GraphML topology before installing an updated version of the same topology.
```

Comment:
```suggestion
IMPORTANT: You must first uninstall a {page-component-title} GraphML topology before installing an updated version of the same topology.
```

Reply:
I suggest not adding the "Important," since this is already under it's own heading and doesn't need to be called out any more than that. Would make sense if it were buried under a lot of other information.

Reply:
My suggestion is to replace the subheading with the IMPORTANT call out. IMO it fits better in the callout rather than a subheading for Notes (plural) when there really is only one.

Reply:
Yes. You're right. I missed that you had removed the title. @mfuhrmann , please ignore my comment, and go with Mark's suggestion. It makes more sense to get rid of the heading and use IMPORTANT.